backends/dart_dia/README.md

```diff
@@ -0,0 +1,13 @@
+# How to run?
```

Comment:
I think we should also add the link to documentation for the packages as a lot of people might discover new packages from this project it might help them with docs.